fenix/app/src/main/java/org/mozilla/fenix/compose/Menu.kt

```diff
@@ -62,14 +69,50 @@ private fun Menu(
                 .background(color = FirefoxTheme.colors.layer2)
                 .then(modifier),
         ) {
+            val hasCheckedItems = menuItems.any { it.isChecked }
+
             for (item in menuItems) {
+                val modifierCheckMark = if (hasCheckedItems) {
+                    Modifier.selectable(
+                        selected = item.isChecked,
+                        role = Role.Button,
+                        onClick = {
```

Comment:
Yes, we need it because onClick is a mandatory parameter, and if I leave the method onClick empty when TaskBack is activated, the click will not work.